bash_tools/sgug_bashrc

```diff
@@ -0,0 +1,58 @@
+sgcp() {
+        cp -r ~/rpmbuild/sgug-rse/packages/"$1"/* ~/rpmbuild/
+}
+sgwipcp() {
+        cp -r ~/rpmbuild/sgug-rse-wip/packages/"$1"/* ~/rpmbuild/
+}
+sgsbuild() {
+        cd ~/rpmbuild/SPECS
+        TARGET="$(ls -1 ~/rpmbuild/SRPMS/ | grep ${1})"
+        rpm -ivh ~/rpmbuild/SRPMS/${TARGET}
+}
+sgrbuild() {
+        rpmbuild --undefine=_disable_source_fetch -ba ${1}.spec --nocheck
+}
+sggit()  {
+        cd ~/rpmbuild/sgug-rse
+}
+sgspec() {
+        cd ~/rpmbuild/SPECS
+}
+sgsrc()  {
+        cd ~/rpmbuild/SRPMS
+}
+sgwip()  {
+        cd ~/rpmbuild/sgug-rse-wip
```

Comment:
sgug-rse-wip is no longer a separate repository. I can merge it like this, but maybe you want to check it first?